.github/workflows/daily.yml

```diff
@@ -34,7 +34,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ["ubuntu-latest", "windows-latest", "macos-latest"]
+        # As of 2024-10-18, ubuntu-latest can refer to different Ubuntu versions,
+        # which can can cause problems with os module constants.
+        os: ["ubuntu-latest-24.04", "windows-latest", "macos-latest"]
```

Comment:
That's not a valid Ubuntu version — we could be waiting for a while for some  runners to match :-)

```suggestion
        os: ["ubuntu-24.04", "windows-latest", "macos-latest"]
```

Reply:
Ha, whoops.